editor/src/components/inspector/controls/option-chain-control.tsx

```diff
@@ -45,6 +45,7 @@ export const OptionChainControl: React.FunctionComponent<DEPRECATEDControlProps<
       display: 'flex',
       flexDirection: 'column',
       marginBottom: 0,
+      width: '100%',
```

Comment:
Were these changes intended as part of this?

Reply:
yes, the align-items control is now in a contextmenu wrapper and it shrinked in size.